rust/processor/src/db/common/models/token_v2_models/parquet_v2_token_datas.rs

```diff
@@ -0,0 +1,256 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+// This is required because a diesel macro makes clippy sad
+#![allow(clippy::extra_unused_lifetimes)]
+#![allow(clippy::unused_unit)]
+
+use crate::{
+    bq_analytics::generic_parquet_processor::{GetTimeStamp, HasVersion, NamedTable},
+    db::common::models::{
+        object_models::v2_object_utils::ObjectAggregatedDataMapping,
+        token_models::token_utils::TokenWriteSet,
+        token_v2_models::{
+            v2_token_datas::CurrentTokenDataV2,
+            v2_token_utils::{TokenStandard, TokenV2, TokenV2Burned},
+        },
+    },
+    utils::util::standardize_address,
+};
+use allocative_derive::Allocative;
+use anyhow::Context;
+use aptos_protos::transaction::v1::{DeleteResource, WriteResource, WriteTableItem};
+use bigdecimal::ToPrimitive;
+use field_count::FieldCount;
+use parquet_derive::ParquetRecordWriter;
+use serde::{Deserialize, Serialize};
+
+#[derive(
+    Allocative, Clone, Debug, Default, Deserialize, FieldCount, ParquetRecordWriter, Serialize,
+)]
+pub struct TokenDataV2 {
+    pub txn_version: i64,
+    pub write_set_change_index: i64,
+    pub token_data_id: String,
+    pub collection_id: String,
+    pub token_name: String,
+    pub largest_property_version_v1: Option<u64>,
+    pub token_uri: String,
+    pub token_properties: String,
+    pub description: String,
+    pub token_standard: String,
+    pub is_fungible_v2: Option<bool>,
```

Comment:
@rtso what's the diff between token_standard and is_fungible_v2?

Reply:
`token_standard` is whether the token is v1 or v2. `is_fungible_v2` is whether this token is also fungible (it's an implementation detail based on the contract)